HealthItem/src/test/java/com/github/sef24sp4/healthitem/HealthItemTest.java

```diff
@@ -0,0 +1,62 @@
+package com.github.sef24sp4.healthitem;
+
+import com.github.sef24sp4.common.entities.ICollidableEntity;
+import com.github.sef24sp4.common.interfaces.IEntityManager;
+import com.github.sef24sp4.common.item.ItemRarity;
+import com.github.sef24sp4.common.metadata.GameElementType;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class HealthItemTest {
+	@Mock
+	private IEntityManager mockEntityManager = mock(IEntityManager.class);
+
+	@Mock
+	private ICollidableEntity mockEntity = mock(ICollidableEntity.class);
```

Comment:
When using the `@Mock` annotation you should omit the instantiation, and instead use the `@ExtendsWith(MockitoExtension)` annotation. [Example](https://github.com/SE-F24-SP4/semesterproject-4/blob/a44129194675e5a05197e91b019856dd4f2e58f3/CollisionSystem/src/test/java/com/github/sef24sp4/collisionsystem/CollisionSystemTest.java#L20-L25)